src/dotnet/Mgmt.CI.BuildTools/CI/CI.Common/Mgmt.CI.Common/Services/ReflectionService.cs

```diff
@@ -230,5 +242,45 @@ string GetNameOfCoreAssembly()
             return typeof(object).Assembly.GetName().Name;
         }
         #endregion
+
+        #region assembly
+        public static readonly byte[] s_SimpleNameOnlyImage = Convert.FromBase64String(
```

Comment:
What is this?  It doesn't appear to be used as you commented out the line that used it.

Reply:
This is my attempt to use reflection in .net core scenarios and this was one way to use system.privatecorelib.dll and use the metadata assembly to satisfy assembly loading. So kept it if I ever need it again.